custom/icds_reports/models.py

```diff
@@ -31,7 +31,7 @@ class Meta:
 
 
 class AggAwcDailyView(models.Model):
-    awc_id = models.TextField(primary_key=True)
+    awc_id = models.TextField(blank=True, null=True)
```

Comment:
I don't htink these can be blank or null. have you seen that?

Reply:
I generated models by ```manage.py inspectdb```